### ISSUE TITLE
"Additional MPV options" not being set (tscale, interpolation, video-sync)

### ISSUE THREAD

**Issue by Smithy3** [lhc70000/iina#1718]
**System and IINA version:**

- macOS 10.13.4
- IINA 0.0.15.1

**Expected behavior:**
With the following options
```
video-sync=display-resample
interpolation=yes
tscale=triangle
```
I would expect panning shots to be very smooth.

**Actual behavior:**
The options are clearly not being set as it looks just the same with/without those options. 
I have checked in the inspector and they are listed under the "Watch" section of "Status". 
The software and system have been restarted several times and the problem still persists.
I have also tried loading them in from an mpv.conf file instead and it still doesn't change anything.
MPV does not have this issue and panning shots are very obviously smoother when comparing the same file in both players. The only lines contained in the mpv config file were those listed above.

**Steps to reproduce:**
1) Delete all options in additional MPV options.
2) Play a video file with a panning shot. Example file [here](https://nyaa.si/view/887190). Roughly 0:00 to 0:18 of episode 1.
3) Insert three lines listed above into the additional options section.
4) Restart IINA.
5) Open up video.
6) Go into inspector and add them the options under "Status".
7) Restart IINA again.
8) Play the same video file.
9) Panning shot should look smoother but appears to look the same.

- [x] MPV does not have this problem.

**How often does this happen?**
All the time.

**Additional note:**
Maybe its a problem with the way that I am setting the options, but I thought I'd create an issue anyway as I don't see how it would be done in another way.

**Comment by inflation** [comment]
`video-sync` currently have some bugs because we do not report FPS back to mpv. If you manually set it, you would hit another bug at #741 and #1542. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by lhc70000** [comment]
There's already a working fix on `mpv-new-api` branch (see #1542). It should be available once mpv 0.29 is released.

**Comment by Smithy3** [comment]
Great, I look forward to the next IINA version.
